misc/trustee-operator-install/install.sh

```diff
@@ -355,5 +369,9 @@ apply_operator_manifests
 
 wait_for_deployment trustee-operator-controller-manager trustee-operator-system || exit 1
 
+# Change trustee image
+override_trustee_image
```

Comment:
Thanks @lmilleri for this PR. Do you mind putting this behind a conditional ?
For example 
if [ "$TRUSTEE_IMAGE" != "" ]; then
   override_trustee_image
fi

Also updating the `display_help()` function to mention about TRUSTEE_IMAGE